frontend/src/pages/DevicesPage/DevicesPage.tsx

```diff
@@ -11,6 +11,7 @@ import { IDevice } from 'remote.it'
 import { Icon } from '../../components/Icon'
 import styles from '../../styling'
 import analytics from '../../helpers/Analytics'
+import { emit } from '../../services/Controller'
```

Comment:
Remove

Reply:
I kept this one. I'd like to have it on the connections page, devices page and settings page. So we have a better chance of it getting updated when a user comes back to the app. If you have another idea for this let me know.

Reply:
Lets just do the heartbeat hook. That should cover everyone and only run when oob is available so won't impact most users.